packages/p2p/src/components/my-profile/payment-methods/add-payment-method/add-payment-method.scss

```diff
@@ -38,6 +38,19 @@
                 margin-top: auto;
                 padding-top: 1.6rem;
             }
+
+            &--fixed-footer {
+                background: var(--general-main-1);
+                border-bottom-left-radius: 8px;
+                border-bottom-right-radius: 8px;
```

Comment:
Instead of `border-radius`, added `border-bottom-right-radius` and `border-bottom-left-radius` so that the bottom border is only applied with `8px` radius:

### Before:
<img width="624" alt="Screenshot 2022-07-27 at 11 53 58 AM" src="https://user-images.githubusercontent.com/103016120/181187408-cdf5c460-b37b-4f83-ad45-5b6075229af9.png">

### After:
<img width="574" alt="Screenshot 2022-07-27 at 3 26 48 PM" src="https://user-images.githubusercontent.com/103016120/181187495-fee23807-8272-4974-b8ef-fbbef1873254.png">